src/App.js

```diff
@@ -86,15 +93,19 @@ class App extends Component {
             results: [
               {
                 slug: "dashboard2",
+                url: "https://nxt3.staging.lizard.net/dashboard/dashboard2",
```

Comment:
The problem is that the slug is needed to call the bootstrap api:
/bootstrap/slug
this can be proxied from localhost:3000 to nxt3.staging.lizard.net.

The link to the complete dashboard cannot be proxied (as far as I know) because html with complete pagerefreshes does not work in the proxy. The url is thus saved here so that we don not have to proxy it on dev, but can just use it at is.
It is a bit dirty, but it does allow us to test the url link even on dev.